packages/atlas/src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -426,7 +426,15 @@ export const CreateEditChannelView: FC<CreateEditChannelViewProps> = ({ newChann
           render={({ field: { value } }) => (
             <>
               <ChannelCover
-                assetUrl={loading ? null : value.croppedUrl || value.originalUrl}
+                assetUrls={
```

Comment:
I'm a big fan of large ternaries, but I think its not the best codestyle tbh. @drillprop  thoughts?